communities/plymouth-university/community-assessment.md

```diff
@@ -0,0 +1,122 @@
+
+## Accessibility
+
+### How accessible are the current resources in your community?
+
+* Hardware is readily available to any student at the University though much of the VR kit is often only taken by Game Dev students.
+* All campus locations have wheelchair access to every room.
+* GDSoc events are often limited to members. Larger events like guest speakers are often open to all at the University with priority to GDSoc members, Game Jams are the same.
+
+### Are meetings held during hours where people may be working?
+
+Most events take place in the evenings, the only exceptions are guest speakers where the organisers always try to suit the speakers time preferences. Game Jams are generally held at weekends to allow sleep catch up.
+
+### Is there childcare available?
+
+No, this isn't currently within the means of GDSoc and the rest of the campus community is fairly disperesed.
+
+## Interests
+
+As the university currently only has one tech based society, most of this section will be dictate by that society with a view to attact new members from across the university.
+
+### Is your community made up only of computer science students? Are you OK with that?
+
+GDSoc is made up of computer scientists and aritsts, there is a large imbalance between engineers and designers within the society. However as it is often run by members of the Computing & Games Dev course it's content is often more tailored towards engineers.
+
+I'd love to get more designers involved and also some first time programmers who aren't on comp sci courses!
+
+### Is your community open to other topics, or are their interests very specific?
+
+If we take the only tech society on campus then the interest is specifically games development, but the university as a whole has many computing and comp sci course flavours who have a wide range of interests from web to AI and all sorts.
+
+## Goals
+
+### Why do people get involved in the community?
+
+One of the biggest across the university seems to be obtaining a 1 year internship for their "sandwich year", or for getting a job after their degree. Due to the games students being most involved, they also seem to get move involved when the content is likely to contribute towards current assignments.
+
+The one of the problem with GDSoc is that most of the Games Dev course content is based on Unity3D, however this is quite an uncommon technology to use outside of mobile or indie games. Web content seems to cause heavy disengagement from GDSoc members as there seems to be a level of elitisim that attempts to stray away from web based games.
+
+### What activities and goals can we set for the community based on existing goals?
+
+One of the goals we could look towards is trying to find to put on technical talks by experts which relate heavily to lecture content. This has been hard due to the lack of industry in and around Plymouth but it should be a goal to strive towards.
+
+## Location
```

Comment:
Space is totally a non-issue thankfully, it's just filling it! The biggest event we had this year was a talk by a friend of mine at JustEat who provided free chicken for everyone haha